src/Controls/tests/DeviceTests/Elements/Frame/FrameTests.cs

```diff
@@ -302,7 +302,7 @@ public async Task FrameIncludesPadding()
 			Assert.Equal(expected, layoutFrame.Height, 1.0d);
 		}
 
-#if !ANDROID
+#if !ANDROID && !IOS
```

Comment:
This test deals with window resizing which isn't applicable to iOS. I updated the iOS tests a bit on this PR to always run inside a Modal View Controller. Because of that change these tests started failing because the iOS view isn't going to change size when you change the width/height on the window